src/containers/PatientDetails/Dashboard/config.ts

```diff
@@ -0,0 +1,139 @@
+import { Patient } from 'fhir/r4b';
+import moment from 'moment';
+
+import { formatFHIRDate } from '@beda.software/fhir-react';
+
+import { DashboardInstance } from 'src/components/Dashboard/types';
+import {
+    prepareActivitySummary,
+    prepareAllergies,
+    prepareConditions,
+    prepareConsents,
+    prepareImmunizations,
+    prepareMedications,
+    prepareSeriveRequest,
+} from 'src/containers/PatientDetails/PatientOverviewDynamic/components/StandardCard/prepare';
+import { PrepareFunction } from 'src/containers/PatientDetails/PatientOverviewDynamic/components/StandardCard/types';
+import { AppointmentCardContainer } from 'src/containers/PatientDetails/PatientOverviewDynamic/containers/AppointmentCardContainer';
+import { CreatinineDashoboardContainer } from 'src/containers/PatientDetails/PatientOverviewDynamic/containers/CreatinineDashoboardContainer';
+import { GeneralInformationDashboardContainer } from 'src/containers/PatientDetails/PatientOverviewDynamic/containers/GeneralIInformationDashboardContainer';
+import { StandardCardContainerFabric } from 'src/containers/PatientDetails/PatientOverviewDynamic/containers/StandardCardContainerFabric';
+
+export const patientDashboardConfig: DashboardInstance = {
```

Comment:
@atuonufure dashboard layout should match the old one.
Also, please delete the old dashboard and use just the dynamic one.